YACS-RCOS/yacs.n
# Resolved Default Semester Bug

**Issue**
Closes #489

**Database Changes/Migrations**
Will delete any previous default semesters before adding/update current default semester (`admin_settings.semester`).
No new migrations.

**Test Modifications**
Not yet.

**Test Procedure**
> 1. Load 2 semester data. The first semester uploaded should be the default semester
> 2. Go into Admin panel, change default semester to the 2nd uploaded semester
> 3. ~~YACS still has the first semester as the default semester~~
3. YACS now should correctly update the default semester

**Photos**
None ( ° ∀ ｡ ).

**Additional Info**
The cache for `GET /api/defaultsemester` is disabled.
